ldes-server-maintenance/ldes-server-compaction/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/compaction/batch/CompactionStepDefinitions.java

```diff
@@ -0,0 +1,42 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.compaction.batch;
+
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.partition.support.Partitioner;
+import org.springframework.batch.core.repository.JobRepository;
+import org.springframework.batch.core.step.builder.StepBuilder;
+import org.springframework.batch.core.step.tasklet.Tasklet;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.task.SimpleAsyncTaskExecutor;
+import org.springframework.transaction.PlatformTransactionManager;
+
+@Configuration
+public class CompactionStepDefinitions {
+	public static final String COMPACTION_STEP = "partitionedCompactionStep";
```

Comment:
Would personally drop the partitioned. Does not provide extra info when reading jobs/steps